pkg/security/serializers/serializers.go

```diff
@@ -257,6 +272,8 @@ type EventContextSerializer struct {
 	Outcome string `json:"outcome,omitempty"`
 	// True if the event was asynchronous
 	Async bool `json:"async,omitempty"`
+	// The list of rules that the event matched (only valid in the context of an anomaly)
+	MatchedRules []*MatchedRuleSerializer `json:"matched_rules"`
```

Comment:
should this be `omitempty` since we may never fill it

Reply:
why a pointer `*MatchedRuleSerializer` and not directly a `MatchedRuleSerializer` ?